ard_controller/ard_controller.ino

```diff
@@ -0,0 +1,29 @@
+#include <DueTimer.h>
+
+bool write_flag = false;
+
+void setup() {
+  // put your setup code here, to run once:
+  Serial.begin(115200);
+  Serial1.begin(115200);
+
+  Timer1.attachInterrupt(serial_ISR).setFrequency(1).start();
```

Comment:
just another question out of curiosity: does this function just repeat the pings until a pong responds once?

Reply:
It sends a ping every second regardless of if a pong is received

Reply:
ah okok makes sense